app/src/main/java/com/teamltt/carcare/database/DbHelper.java

```diff
@@ -141,6 +141,24 @@ public long createNewTrip(long vehicleId, Date startTime, Date endTime) {
         return status;
     }
 
+    public long createNewUser(String google_user_id, String firstName, String lastName) {
+        SQLiteDatabase db = getWritableDatabase();
+        long status = DbHelper.errorChecks(db);
+        if (status != DbHelper.DB_OK) {
+            return status;
+        }
+        status = UserContract.insert(db, google_user_id, firstName, lastName);
+        db.close();
+        return status;
+    }
+
+    public boolean containsUser(String google_user_id) {
+        SQLiteDatabase db = getReadableDatabase();
+        Cursor cursor = UserContract.queryUserID(db, google_user_id);
+        db.close();
+        return cursor != null;
```

Comment:
I believe so. The cursor should be null if no user with the given id is found. If you can tell me how to check the database I'll make sure it's not duplicated.